contents/tutorials/nuxt-surveys.md

```diff
@@ -0,0 +1,553 @@
+import ImgSurveyTemplatesLight from '../images/tutorials/nuxt-surveys/survey-templates-light.png'
+import ImgSurveyTemplatesDark from '../images/tutorials/nuxt-surveys/survey-templates-dark.png'
+
+[Surveys](/docs/surveys) are a great way to get feedback from your users. In this guide, we show you how to add a survey to your Nuxt.js app.
+
+We'll create a basic Nuxt app, add PostHog, create a survey, and then show you how to display the survey in the app and get responses.
+
+## 1. Create a Nuxt app
+
+For this tutorial, we create a basic `Nuxt 3` app. First, ensure [Node.js is installed](https://nodejs.dev/en/learn/how-to-install-nodejs/) (version 14.6.0 or newer). Then run the following command:
+
+```bash
+npx nuxi@latest init <project-name>
+```
+
+Name it whatever you like (we call ours `nuxt-surveys`), select `npm` as the package manager, and use the defaults for the remaining options.
+
+Replace the code in `app.vue` with a simple heading and button:
+
+```vue file=app.vue
+<template>
+  <main>
+    <h1>Nuxt.js 3 Surveys</h1>
+  </main>
+</template>
+```
+
+Run `npm run dev` to start your app.
+
+![Basic app](../images/tutorials/nuxt-surveys/basic-app.png)
+
+## 2. Add PostHog
+
+> This tutorial shows how to integrate PostHog with `Nuxt 3`. If you're using `Nuxt 2`, see [our Nuxt docs](/docs/libraries/nuxt-js) for how to integrate PostHog.
+
+We use PostHog to create and control our survey as well as monitor results. If you don't have a PostHog instance, you can [sign up for free here](https://us.posthog.com/signup). 
+
+First install `posthog-js`:
+
+```bash
+npm install posthog-js
+```
+
+Then, add your PostHog API key and host to your `nuxt.config.ts` file. You can find your project API key in your [PostHog project settings](https://app.posthog.com/settings/project)
+
+```ts file=nuxt.config.ts
+export default defineNuxtConfig({
+  // ...rest of your config
+  
+  runtimeConfig: {
+    public: {
+      posthogPublicKey: '<ph_project_api_key>',
+      posthogHost: '<ph_instance_address>'
+    }
+  }
+})
+```
+
+Create a new [plugin](https://nuxt.com/docs/guide/directory-structure/plugins) by first creating a new folder called `plugins` and then a new file `posthog.client.js`:
+
+```bash
+mkdir plugins
+cd plugins 
+touch posthog.client.js
+```
+
+Add the following code to your `posthog.client.js` file:
+
+```js file=plugins/posthog.client.js
+import { defineNuxtPlugin } from '#app'
+import posthog from 'posthog-js'
+
+export default defineNuxtPlugin(nuxtApp => {
+  const runtimeConfig = useRuntimeConfig();
+  const posthogClient = posthog.init(runtimeConfig.public.posthogPublicKey, {
+    api_host: runtimeConfig.public.posthogHost,
+  })
+
+  // Make sure that pageviews are captured with each route change
+  const router = useRouter();
+  router.afterEach((to) => {
+    posthog.capture('$pageview', {
+      current_url: to.fullPath
+    });
+  });
```

Comment:
Do we need this for this tutorial? Even if it acts as a SPA, should capture at least an initial pageview on first load to prove it is working.

If you don't remove it, should at `capture_pageview: false` to initialization config.

Reply:
Yeah fair enough. Will remove